bindings/hyperdrivepy/python/hyperdrivepy/pypechain_types/IHyperdriveContract.py

```diff
@@ -636,6 +636,33 @@ def call(
         return cast(bool, rename_returned_types(structs, return_types, raw_values))
 
 
+class IHyperdriveKindContractFunction(ContractFunction):
```

Comment:
This is generated by pypechain, right? If so, what is the benefit of committing these files? For contract wrappers in other languages, we gitignore wrappers since they are built during or before compilation

Reply:
Yes, they're generated by pypechain. I am not sure who made that call (could have been me) and I don't recall why. I think what you're saying makes sense, though. I will look into it when I have a chance.